src/components/Footer/__test__/Footer.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from "react-test-renderer";
+import Footer from "../Footer";
```

Comment:
About the warnings .. we can change the `lint.yml` to run `lint:fix` instead of `lint:strict` 
I think this way it fixes the warning instead of printing them, what do you say @Lilicod  ?